cfme/tests/storage/test_manager.py

```diff
@@ -11,13 +12,19 @@
     pytest.mark.tier(3),
     test_requirements.storage,
     pytest.mark.usefixtures('setup_provider'),
-    pytest.mark.provider([OpenStackProvider])
+    pytest.mark.provider([OpenStackProvider], scope='module')
 ]
 
 
 MANAGER_TYPE = ['Swift Manager', 'Cinder Manager']
 
 
+@pytest.yield_fixture(scope='module')
```

Comment:
**REQUIRED:** I would prefer to see the ```openstack_provider``` object being used in the fixture rather than clearing all CloudProviders. Is that possible?

Reply:
@psav why not. thank's for pointing this.